src/components/MessagesRow.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import styles from '../styles/styles';
+import Icon from './Icon';
+import * as Expensicons from './Icon/Expensicons';
+import DotIndicatorMessage from './DotIndicatorMessage';
+import Tooltip from './Tooltip';
+import CONST from '../CONST';
+import * as StyleUtils from '../styles/StyleUtils';
+import PressableWithoutFeedback from './Pressable/PressableWithoutFeedback';
+import stylePropTypes from '../styles/stylePropTypes';
+import useLocalize from '../hooks/useLocalize';
+
+const propTypes = {
+    // The error messages to display
+    messages: PropTypes.objectOf(PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.object]))])),
+
+    // The type of message, 'error' shows a red dot, 'success' shows a green dot
+    type: PropTypes.oneOf(['error', 'success']).isRequired,
+
+    /** A function to run when the X button next to the message is clicked */
+    onClose: PropTypes.func,
+
+    /** Additional style object for the container */
+    containerStyles: stylePropTypes,
+
+    /** Whether we can dismiss the error message */
+    canDismissError: PropTypes.bool,
```

Comment:
Good point. I copied this straight in from OfflineWithFeedback. So why did we add it there? @luacmartins added it in [this PR](https://github.com/Expensify/App/pull/29718) so maybe he can do a refactor if we think it's important.

Having a `canDismissError` prop is more explicit. If not we would have to check if the `onClose` function was set to the default noop, which we could do but it's a bit weird. I think it's ok as is.

I'm renaming it in this component to `canDismiss` since it's not specific to errors now.